ci-operator/step-registry/cucushift/installer/check/nutanix/multi-disks/cucushift-installer-check-nutanix-multi-disks-ref.metadata.json

```
@@ -0,0 +1,15 @@
+{
+	"path": "cucushift/installer/check/nutanix/multi-disks/cucushift-installer-check-nutanix-multi-disks-ref.yaml",
```

```suggestion
	"path": "cucushift/installer/check/nutanix/compute/data-disks/cucushift-installer-check-nutanix-compute-data-disks-ref.yaml",
```
I guess we need to update the step name accordingly.